collectd_opentsdb_plugin.py

```diff
@@ -0,0 +1,62 @@
+import os
+import potsdb
+import collectd
+
+
+class OpenTSDBExportPlugin:
+
+    def __init__(self):
+        self.metrics = None
+
+    def configure_callback(self, conf):
+        for node in conf.children:
+            key = node.key.lower()
+            value = node.values[0]
+            if key == 'host':
+                self._opentsdb_host = value
+            elif key == 'port':
+                self._opentsdb_port = value
+        if not self._opentsdb_host or not self._opentsdb_port:
+            raise Exception("OpenTSDB export plugin is not configured")
+        collectd.info("Configured OpenTSDB export plugin (%s:%s)" %
+                      (self._opentsdb_host, self._opentsdb_port))
+
+    def init_callback(self):
+        self.metrics = potsdb.Client(host=self._opentsdb_host,
+                                     port=self._opentsdb_port,
+                                     mps=100, check_host=True)
```

Comment:
That's pretty important thing to consider in advance, I'm afraid that the plugin fail doesn't mean collectd fail, it will continue to work (just w/o this plugin) and this means that Marathon restart system is useless here.

From the other side, if we disable `check_host` - it can lead to fast grow of that potsdb queue (N job * 20 metrics per 1 second). But it's OK because `potsdb` will just start drop first metrics when the queue is full (https://github.com/scrapinghub/potsdb/blob/master/potsdb/client.py#L151).

Also for notes, a potsdb PR for backoff in retries https://github.com/orionvm/potsdb/pull/4. I hope that we don't need to keep our own fork of the client.